build/kactlprocessor.py

```diff
@@ -128,6 +129,10 @@ def processwithcomments(caption, instream, outstream, listingslang = None):
         nsource = nsource.rstrip() + source[end:]
     nsource = nsource.strip()
 
+    hash_script = 'hash' if listingslang else 'hash-cpp'
```

Comment:
maybe skip hashing for .txt, .sh and .py?

Reply:
We already don't do hashing for those, since (at least for .sh and .txt), they're not processed by `processwithcomments`, they're processed by `processraw`.

Regardless, in according with the other comment, I'll probably just hash everything with `hash-cpp`.